src/query/config/src/setting.rs

```diff
@@ -57,18 +57,18 @@ pub struct Config {
     // external catalog config.
     // - Later, catalog information SHOULD be kept in KV Service
     // - currently only supports HIVE (via hive meta store)
-    pub catalogs: HashMap<String, CatalogConfig>,
+    pub catalogs: HashMap<String, CatalogSetting>,
 
     // Cache Config
-    pub cache: TheCache,
+    pub cache: CacheSetting,
 }
 
-impl Config {
-    /// As requires by [RFC: Config Backward Compatibility](https://github.com/datafuselabs/databend/pull/5324), we will load user's config via wrapper [`ConfigV0`] and then convert from [`ConfigV0`] to [`Config`].
+impl Setting {
```

Comment:
The name of setting may confused with session settings, why change the name?

Reply:
By design, Config can't be update when running but setting can be.

Reply:
The former `Outer` `Inner` naming is rather confusing:
the "motivation" of current naming -> 
https://github.com/datafuselabs/databend/blob/0eaf74795ce5ec0379644c186e115c1932d6cc28/src/query/config/src/lib.rs#L23-L25